documentation/typedoc.json

```diff
@@ -1,15 +1,13 @@
 {
   "excludeExternals": true,
-  "externalPattern": ["../viewer/src/__tests__/**"],
+  "externalPattern": ["../viewer/src/__tests__/**", "../viewer/node_modules/**"],
   "exclude": ["**/*+(.spec|.test).ts"],
   "excludePrivate": true,
   "excludeProtected": true,
-  "stripInternal": true,
+  "excludeInternal": true,
```

Comment:
Is there a semantic difference here, or simply a rename?

Reply:
yeah, reference to a doc or just common text in the PR description would be great to quickly understand the reasons behind changes 🙂

Reply:
Sorry, it's removed and picked up from `tsconfig.json` but I have it here to be "sure"... See https://github.com/TypeStrong/typedoc/issues/1469